### PR TITLE
feat(UI): form to input an activation code

### PR DESCRIPTION
**Pull request by xcoulon**
Adding an input field for an activation code, which triggers the approval of the user (if valid), and forwards to the usual page where users can access the Web Console

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Comment by xcoulon** [comment]
image

File: pkg/verification/service/verification_service.go [hunk 1]
```diff
@@ -393,7 +393,7 @@ func (s *ServiceImpl) VerifyActivationCode(ctx *gin.Context, userID, username, c
 	}
 	// if there is room for the user and if the "time window" to signup is valid
 	now := metav1.NewTime(time.Now())
-	log.Infof(ctx, "verifying activation code '%s': event.Status.ActivationCount=%d, event.Spec.MaxAttendees=%d, event.Spec.StartTime=%s, event.Spec.EndTime=%s", code, strconv.Itoa(event.Status.ActivationCount), strconv.Itoa(event.Spec.MaxAttendees), event.Spec.StartTime.Format("2006-01-02:03:04:05"), event.Spec.EndTime.Format("2006-01-02:03:04:05"))
+	log.Infof(ctx, "verifying activation code '%s': event.Status.ActivationCount=%d, event.Spec.MaxAttendees=%s, event.Spec.StartTime=%s, event.Spec.EndTime=%s", code, strconv.Itoa(event.Status.ActivationCount), strconv.Itoa(event.Spec.MaxAttendees), event.Spec.StartTime.Format("2006-01-02:03:04:05"), event.Spec.EndTime.Format("2006-01-02:03:04:05"))
```
Comment (alexeykazakov) [inline pkg/verification/service/verification_service.go:396]: You sure you want to use %s?

**Review by alexeykazakov — APPROVED** [review]

File: pkg/verification/service/verification_service.go (hunk 1 above)
Comment (xcoulon) [reply pkg/verification/service/verification_service.go:396]: well, the arg is `strconv.Itoa(event.Spec.MaxAttendees)` so yes, but I wonder why I even started with `strconv.Itoa(event.Spec.MaxAttendees)` and not just `event.Spec.MaxAttendees` 🙄 🤔 

And same question for `strconv.Itoa(event.Status.ActivationCount)` vs `event.Status.ActivationCount`!

**Pull request merged**